src/workerd/io/worker-entrypoint.c++

```diff
@@ -338,6 +338,7 @@ kj::Promise<void> WorkerEntrypoint::request(kj::HttpMethod method,
     loggedExceptionEarlier = true;
     context.logUncaughtExceptionAsync(UncaughtExceptionSource::REQUEST_HANDLER, kj::cp(exception));
 
+    // TODO(now): Can we just drop this?
```

Comment:
This reportFailure call was causing some kind of failures so I commented it out previously. Not sure if this is still needed in the new model now that exception and outcome reporting go through WorkerTracer.